conf/application.conf

```diff
@@ -108,7 +108,7 @@ microservice {
         port = 9570
         max-uploads = 2
         max-file-size = 3145728
-        s3-url-expiry-duration = 5days
+        s3-url-expiry-duration = 5seconds
```

Comment:
this should be changed back to the default 5days

Reply:
Done. Very sorry.